### PR TITLE
Content clarity

### PR DESCRIPTION
**Pull request by alexandrascript**
This PR cleans up the content for clarity. This includes:

+ Defining the terms "attribute" and "method" for non-technical audiences
+ "Model" is now always known as "attribute" 
+ Clean up the key concepts list to clearly define the term and then give examples
+ Clarify the difference between a model and a model attribute
+ Define what a limit is and why it exists

I've also fixed a couple old / broken links.

### CODE REVIEW

File: 2-api/limits.md [hunk 1]
```diff
@@ -2,6 +2,10 @@
 
 # Limits and errors
 
+The Perspective API uses limits and quotas to protect the system from receiving more data than it can handle, and to ensure equitable distribution of the system resources. These limits and quotas are subject to change.
+
+You may encounter an error if the quota of your request exceeds the limit.
```
Comment (oliviaredfield) [inline 2-api/limits.md]: Rephrase to "You may encounter an error if your request exceeds the limit."

Should we be more specific about what the error messages are, if possible?

File: 2-api/methods.md [hunk 2]
```diff
@@ -2,7 +2,9 @@
 
 # API Methods
 
-There are two API methods: 
+A method is a procedure associating a message (or request) with an object (which consists of data and behavior). Once run, a method returns a new object.
```
Comment (oliviaredfield) [inline 2-api/methods.md:5]: This feels too abstract to me. What if instead we have a high level description of what the two different options for requests are? Something like the following (and feel free to machete this):

"There are two methods available for Perspective API, where each method is a different type of request a user can send:
- AnalyzeComment, where a user sends a request for a comment to be analyzed, and a score is returned
- SuggestCommentScore, where a user can suggest a better score fora  comment"

File: 2-api/models.md [hunk 3]
```diff
@@ -2,35 +2,39 @@
...
+Experimental models have not been tested as thoroughly as production models. If using these models, you’ll need to update your code when a model changes from experimental to production. We typically give one month’s notice before deprecating experimental models.
+
+We recommend using experimental models only in non-production environments.
 
 ### Important notes on using experimental models
 
-Experimental versions will be deprecated. These models are experimental and will eventually stop working when the production versions are created, so you will need to update the model name you are calling when the final version is available.
+Experimental models will be deprecated. Once deprecated and once production versions of these models are created, the experimental model will stop working. If that happens, you will need to update the API call's model name to the new production model name.
```
Comment (oliviaredfield) [inline 2-api/models.md]: "Experimental models will be deprecated" - a little ambiguous, maybe rephrase?. In general we will always have experimental models as a category, but each model can and likely will be deprecated, mostly depending on when that model goes into production.

File: 2-api/models.md [hunk 4]
```diff
@@ -2,35 +2,39 @@
...
 
-Experimental versions will make even more mistakes than production models. They may not have been tested for issues including unintended bias and thus it is particularly important that they only be used in contexts where you have a human in the loop to identify and correct errors.
+Experimental versions will make even more mistakes than production models. They may not have been tested for issues, such as unintended bias, and thus it is particularly important that they only be used where a human is tasked with indentifying and correcting errors.
```
Comment (oliviaredfield) [inline 2-api/models.md]: indentifying -> identifying

Also, we don't necessarily expect that they'd send us corrections for errors using SuggestCommentScore. Maybe there is another way to phrase this? This is more so meant to indicate that users should have a human in the loop to review judgments made by experimental models (but really all of our models - we never recommend a fully automated workflow, a human should always be in the loop in some way)

File: 2-api/models.md [hunk 5]
```diff
@@ -2,35 +2,39 @@
 
 # Models and attributes
 
-This section details Perspective API’s production and experimental models, which are described in detail in the [All model types](#all-model-types) table below.
+A model is a set of attributes (sometimes called "model attributes") on which a comment is scored on. For example, the Toxicity model contains attributes such as "Toxcity", "Profanity", "Fliratation", and more. [See all model types](#all-model-types). Models can be in-production or experimental.
```
Comment (oliviaredfield) [inline 2-api/models.md]: Discussed with eng offline - let's remove mention of "models" and just call them attributes, especially since the field is called requestedAttributes in the API request. Then we can avoid having to explain the relationship between these terms. We'll want to update this in a few different sections here, and in the Methods doc as well.

**Review by oliviaredfield — COMMENTED** [review]
Please see inline comments!

File: 2-api/models.md [hunk 6]
```diff
@@ -1,72 +1,70 @@
 [Perspective API documentation](../README.md) > [API Reference Docs](README.md) > **Models and attributes**
 
-# Models and attributes
+# Attributes
 
-A model is a set of attributes (sometimes called "model attributes") on which a comment is scored on. For example, the Toxicity model contains attributes such as "Toxcity", "Profanity", "Fliratation", and more. [See all model types](#all-model-types). Models can be in-production or experimental.
+An attribute (sometimes called "model attributes" or "models") is a label on which a comment is scored. For example, the some attributes scored by Perspective API include as "Toxcity", "Profanity", "Fliratation", and more. [See all attribute types](#all-attribute-types). Attributes can be in-production or experimental.
```
Comment (oliviaredfield) [inline 2-api/models.md]: Remove "model attribute" for simplicity so it's just "sometimes called models" ?

File: 2-api/methods.md [hunk 7]
```diff
@@ -4,10 +4,10 @@
 
 A method is a procedure associating a message (or request) with an object (which consists of data and behavior). Once run, a method returns a new object.
 
-There are two methods available for Perspective API: 
+There are two methods available for Perspective API, where each method is a different type of request a user can send:
 
-+ [`AnalyzeComment`](#scoring-comments-analyzecomment)
-+ [`SuggestCommentScore`](#sending-feedback-suggestcommentscore)
++ [`AnalyzeComment`](#scoring-comments-analyzecomment), where a user sends a request for a comment to be analyzed, and a score is returned
++ [`SuggestCommentScore`](#sending-feedback-suggestcommentscore), where a user can suggest a better score fora comment
```
Comment (oliviaredfield) [inline 2-api/methods.md]: fora -> for a

File: 2-api/clients.md [hunk 8]
```diff
@@ -123,8 +123,8 @@ See the docs [on the project's GitHub page](https://github.com/sloria/perspectiv
 Read more about the Perspective API and how to use it.
 
 * [Key concepts](key-concepts.md)
-* [Models and attributes](models.md)
-   * [Model cards](model-cards/README.md)
+* [Attributes](models.md)
+   * [Attribute cards](model-cards/README.md)
```
Comment (oliviaredfield) [inline 2-api/clients.md:127]: Let's keep this as "model cards", since this is a name we've used in external comms and internal conversations

File: 2-api/README.md [hunk 9]
```diff
@@ -5,8 +5,8 @@
 Read more about the Perspective API and how to use it.
 
 * [Key concepts](key-concepts.md)
-* [Models and attributes](models.md)
-   * [Model cards](model-cards/README.md)
+* [Attributes](models.md)
+   * [Attribute cards](model-cards/README.md)
```
Comment (oliviaredfield) [inline 2-api/README.md]: Keep as "model cards"

File: 2-api/model-cards/README.md [hunk 10]
```diff
@@ -1,4 +1,4 @@
-[Perspective API documentation](../../README.md) >  [API Reference Docs](../README.md) > [Production and experimental models](../models.md) > **Model Cards**
+[Perspective API documentation](../../README.md) >  [API Reference Docs](../README.md) > [Attributes](../models.md) > **Attribute Cards**
```
Comment (oliviaredfield) [inline 2-api/model-cards/README.md]: keep as "model cards"

File: README.md [hunk 11]
```diff
@@ -16,8 +16,8 @@ This repository contains the necessary documentation to start using the Perspect
 1. [API Reference](2-api/)
    + [Key concepts](2-api/key-concepts.md)
    + [Clients](2-api/clients.md)
-   + [Models and attributes](2-api/models.md)
-      + [Model cards](2-api/model-cards/README.md)
+   + [Attributes](2-api/models.md)
+      + [Attribute cards](2-api/model-cards/README.md)
```
Comment (oliviaredfield) [inline README.md]: keep as "model cards"

File: 2-api/limits.md [hunk 12]
```diff
@@ -24,8 +24,8 @@ The maximum text size per request is 3000 bytes.
 Read more about the Perspective API and how to use it.
 
 * [Key concepts](key-concepts.md)
-* [Models and attributes](models.md)
-   * [Model cards](model-cards/README.md)
+* [Attributes](models.md)
+   * [Attribute cards](model-cards/README.md)
```
Comment (oliviaredfield) [inline 2-api/limits.md]: keep as "model cards"

File: 2-api/methods.md [hunk 13]
```diff
@@ -270,8 +270,8 @@ The response is not particularly interesting:
 Read more about the Perspective API and how to use it.
 
 * [Key concepts](key-concepts.md)
-* [Models and attributes](models.md)
-   * [Model cards](model-cards/README.md)
+* [Attributes](models.md)
+   * [Attribute cards](model-cards/README.md)
```
Comment (oliviaredfield) [inline 2-api/methods.md]: keep as "model cards"

File: 2-api/key-concepts.md [hunk 14]
```diff
@@ -2,64 +2,77 @@
...
 # Key concepts
 
-To best understand the reference documentation, you must first know these key concepts.
+To best understand the reference documentation, you must first understand these
+key concepts.
 
 ## Attribute
 
-A comment label used for review. The most popular attributes are the toxicity and severe toxicity models.
+An **attribute** or **model attribute** is a comment label used for review.
```
Comment (oliviaredfield) [inline 2-api/key-concepts.md]: I don't love this definition but don't have a better suggestion at the moment. Is there language we can pull from elsewhere in the documentation that is more clear?

File: 2-api/key-concepts.md [hunk 15]
```diff
@@ -2,36 +2,42 @@
...
 # Key concepts
 
-To best understand the reference documentation, you must first know these key concepts.
+To best understand the reference documentation, you must first understand these key concepts.
 
 ## Attribute
 
-A comment label used for review. The most popular attributes are the toxicity and severe toxicity models.
+An **attribute** is a comment label used for review. The most popular
+attributes are the toxicity and severe toxicity models.
```
Comment (bfvitale) [inline 2-api/key-concepts.md]: Could we emphasize TOXICITY and SEVERE_TOXICITY in this sentence, since they they are the name of attributes? Also, include the underscore in SEVERE_TOXICITY.

Might be helpful elsewhere in the doc, too, but lemme know what you think.
The style we use for naming attributes seems to be to capitalize them, so perhaps that is the right choice of emphasis here, rather than boldface, initial caps, italics, etc.

Comment (bfvitale) [inline 2-api/key-concepts.md]: perhaps delete the word "models" at the end of this sentence (line 10).

Comment (bfvitale) [inline 2-api/key-concepts.md]: perhaps delete the word "models" at the end of this sentence (line 10).

File: 2-api/key-concepts.md [hunk 16]
```diff
@@ -2,36 +2,42 @@
...
- A **model** is a dimension that the comment is scored on; for example,
- "toxic," "obscene," "thoughtful," "off-topic," etc. This is also sometimes referred to
- as an **attribute** or **model attribute**.
+*(Coming soon)* **Context** is a representation of the conversation context
+for the comment, such as an article that is being commented on, or another
+comment that is being replied to. It is used in the analysis of the
+comment text, such as when determining the "off-topic" model score.
 
-### Model attribute scores
+**Note**: you can send context with your requests, however our current models
```
Comment (bfvitale) [inline 2-api/key-concepts.md]: perhaps go all the way and change "model" to "attribute" here on lines 23, 25 and 26, too.

File: 2-api/key-concepts.md [hunk 17]
```diff
@@ -2,36 +2,42 @@
...
-## Model
+## Context (coming soon)
 
- A **model** is a dimension that the comment is scored on; for example,
- "toxic," "obscene," "thoughtful," "off-topic," etc. This is also sometimes referred to
- as an **attribute** or **model attribute**.
+*(Coming soon)* **Context** is a representation of the conversation context
+for the comment, such as an article that is being commented on, or another
+comment that is being replied to. It is used in the analysis of the
+comment text, such as when determining the "off-topic" model score.
```
Comment (bfvitale) [inline 2-api/key-concepts.md]: rewrite "off-topic" as OFF_TOPIC

File: 2-api/key-concepts.md [hunk 18]
```diff
@@ -2,36 +2,42 @@
...
+scores.
 
-The API can return **model attribute scores** in different formats, known as
-**score types**. Currently, the only score type supported is a probability score
-between 0 and 1, with higher values indicating greater likelihood of the attribute
-label.
+## Model
 
-### Summary Score
+A **model** is a dimension that the comment is scored on; for example,
```
Comment (bfvitale) [inline 2-api/key-concepts.md]: perhaps change model here on 31 to attribute.
Then you could fixup the last sentence (lines 32 / 33) as "This is also sometimes referred to as a **model attribute**.

File: 2-api/key-concepts.md [hunk 19]
```diff
@@ -2,36 +2,42 @@
...
+"toxic," "obscene," "thoughtful," "off-topic," etc. This is also sometimes
```
Comment (bfvitale) [inline 2-api/key-concepts.md]: perhaps change "toxic" to TOXIC, "obscene" to OBSCENE, etc. i.e. removing the quotation marks and capitalizing.
Lemme know if this creates other problems or is not appropriate here.

File: 2-api/key-concepts.md [hunk 20]
```diff
@@ -43,23 +49,24 @@ rest of the comment.
...
 ### Span scores
 
-Model scores for particular subparts of the request's comment. For example, if the API only found one sentence in a paragraph to be "toxic", it could return a high "toxic" span score for the span corresponding to that sentence while giving a low "toxic" span score to the rest of the comment. This score is only available for some models.
+A **span score** is a model score for subparts of the request's comment.
 
-## Context
+For example, if the API only found one sentence in a paragraph to be "toxic",
+it could return a high "toxic" span score for the span corresponding to that
+sentence while giving a low "toxic" span score to the rest of the comment.
+This score is only available for some models.
```
Comment (bfvitale) [inline 2-api/key-concepts.md]: change "models" to "attributes" here (last word sentence of line 57)

File: 2-api/key-concepts.md [hunk 21]
```diff
@@ -43,23 +49,24 @@ rest of the comment.
...
-*(Coming soon)* **Context** is a representation of the conversation context
-for the comment; for example, an article that is being commented on, or
-another comment that is being replied to. It is used in the analysis of the
-comment text; for example, when determining the "off-topic" model score.
+### Summary Score
 
-**Note**: you can send context with your requests, however our current models
-are not making use of it&mdash;so sending context won't change any model's
-scores.
+A model's **summary score** provides an overall score for the entire comment.
```
Comment (bfvitale) [inline 2-api/key-concepts.md]: change "A model's" to "An attribute's"

File: 2-api/key-concepts.md [hunk 22]
```diff
@@ -43,23 +49,24 @@ rest of the comment.
...
+While the API may return multiple span scores for a given input, it will
+always return exactly one summary score.
 
 ## Toxicity
 
-One of the attributes that Perspective can score. Toxicity is defined as "a rude,
-disrespectful, or unreasonable comment that is likely to make you leave a discussion."
+**Toxicity** is an attribute which Perspective can score. Toxicity is defined
```
Comment (bfvitale) [inline 2-api/key-concepts.md]: Perhaps capitalize TOXICITY here. OK to bold it, too, if you like.

**Review by bfvitale — COMMENTED** [review]
looks like a big improvement. I've suggested a few more small changes below.

File: 2-api/clients.md (hunk 8 above)
Comment (rachelrosengoogle) [reply 2-api/clients.md:127]: +1--I think this still needs to be updated (from my view, though github makes seeing changes tricky!)

File: 2-api/key-concepts.md [hunk 23]
```diff
@@ -2,71 +2,84 @@
...
 
-A model's **summary score** provides an overall score for the entire
-comment. While the API may return multiple span scores for a given input, it
-will always return exactly one summary score.
+
+### Score types
+
+**Score types** are different formats for the API to return model attribute
+scores. Currently, the only score type supported is the probability score
+type with a value between 0 and 1. Higher values indicating greater likelihood
```
Comment (rachelrosengoogle) [inline 2-api/key-concepts.md:53]: I don't think this is technically true. Users can request the RAW score, though the results of it aren't very meaningful since they aren't calibrated.

File: 2-api/limits.md [hunk 24]
```diff
@@ -2,6 +2,10 @@
 
 # Limits and errors
 
+The Perspective API uses limits and quotas to protect the system from receiving more data than it can handle, and to ensure equitable distribution of the system resources. These limits and quotas are subject to change.
+
+You may encounter an error if your request exceeds the limit.
+
 ## Quota limit
 
 By default, we set the quota to 1 QPS for all Perspective projects.
```
Comment (rachelrosengoogle) [inline 2-api/limits.md]: Would it be too much to specific that QPS means "query per second" here? It's a commonly used acronym, but I think it's good to clarify acronyms the first time they're used, in case the user hasn't seen them before.

File: 2-api/models.md [hunk 25]
```diff
@@ -1,68 +1,70 @@
-[Perspective API documentation](../README.md) > [API Reference Docs](README.md) > **Models and attributes**
+[Perspective API documentation](../README.md) > [API Reference Docs](README.md) > **Attributes**
 
-# Models and attributes
+# Attributes
 
-This section details Perspective API’s production and experimental models, which are described in detail in the [All model types](#all-model-types) table below.
+An attribute (sometimes called "models") is a label on which a comment is scored. For example, the some attributes scored by Perspective API include as "Toxcity", "Profanity", "Fliratation", and more. [See all attribute types](#all-attribute-types). Attributes can be in-production or experimental.
```
Comment (rachelrosengoogle) [inline 2-api/models.md]: Earlier you wrote that attributes were "previously known as models"; maybe change this here to make this consistent (or change it in the other places to match this)? Or you could say 'sometimes called "model attributes"' which is consistent with other changes in the documentation you've made.

File: 2-api/models.md [hunk 26]
```diff
@@ -90,22 +92,22 @@ These models are experimental because they are trained on a single source of com
...
+Most of our attributes are [Convolutional Neural Network](https://en.wikipedia.org/wiki/Convolutional_neural_network) (CNN) trained with [word-vector](https://www.tensorflow.org/tutorials/word2vec) inputs. You can also train your own [deep CNN for text classification](http://www.wildml.com/2015/12/implementing-a-cnn-for-text-classification-in-tensorflow/) on our [public toxicity dataset](https://figshare.com/articles/Wikipedia_Talk_Labels_Toxicity/4563973), and explore our [open-source attribute training tools](https://github.com/conversationai/conversationai-models) to train your own attributes. Our [Toxicity Kaggle Competition](https://kaggle.com/c/jigsaw-toxic-comment-classification-challenge) also has lots of useful resources to help build your own attributes.
 
-## Model Cards
+## Attribute cards
 
-For each production model, we aim to publish an associated "Model Card" that shares details about intended usage, model training processes, and evaluation results.
+For each production attribute, we aim to publish an associated "Attribute card" that shares details about intended usage, attribute training processes, and evaluation results.
```
Comment (rachelrosengoogle) [inline 2-api/models.md]: I think Olivia mentioned in an earlier comment that we should call these "Model cards" to be consistent with other messaging.

File: 3-concepts/score-normalization.md [hunk 27]
```diff
@@ -32,31 +30,31 @@ of **0.80** can be interpreted as a belief that **80%** of people would consider
...
 better ways to do this, please
 [open an issue on GitHub](https://github.com/conversationai/perspectiveapi/issues).
 
 ## About score feedback
 
-Our models are not perfect and will make errors. It will be unable to detect patterns
-of toxicity it has not seen before, and it will falsely detect comments similar to
-patterns of previous toxic conversations. To help improve the machine learning, the API
-supports sending our team suggested scores and corrections.
+Our attributes are not perfect and will make errors. It will be unable to
```
Comment (rachelrosengoogle) [inline 3-concepts/score-normalization.md]: In this case (and in the paragraph above) it might make more sense to keep model, or say "model attributes" or "machine learning attributes" since when we talk about errors, it's usually in the context of models. Attributes are the *names* we give our model results in the API responses, but it's not the attributes making the errors, it's the models.

File: 5-contribute/label-data.md [hunk 28]
```diff
@@ -7,8 +7,8 @@ but you can also contribute data from your own community.
 
 ### Why contribute labeled data
 
-Submitting labeled data lets the Perspective API team train the models on more
-comments, and makes it possible for the API to improve and learn from your
+Submitting labeled data lets the Perspective API team train the attributes on
```
Comment (rachelrosengoogle) [inline 5-contribute/label-data.md]: Same thought as comment above--we train models, not attributes, so this might be a good place to say "model attributes" to be clearer,

**Review by rachelrosengoogle — COMMENTED** [review]
Apologies for the delay in reviewing this!

File: 2-api/key-concepts.md (hunk 22 above)
Comment (alexandrascript) [reply 2-api/key-concepts.md]: 👍

File: 2-api/key-concepts.md (hunk 21 above)
Comment (alexandrascript) [reply 2-api/key-concepts.md]: 👍

File: 2-api/limits.md (hunk 24 above)
Comment (alexandrascript) [reply 2-api/limits.md]: Absolutely!

File: 3-concepts/score-normalization.md [hunk 29]
```diff
@@ -44,11 +44,11 @@ better ways to do this, please
 
 ## About score feedback
 
-Our attributes are not perfect and will make errors. It will be unable to
-detect patterns of toxicity it has not seen before, and it will falsely detect
-comments similar to patterns of previous toxic conversations. To help improve
-the machine learning, the API supports sending our team suggested scores and
-corrections.
+Our machine learning attributes are not perfect and will make errors. It will
```
Comment (rachelrosengoogle) [inline 3-concepts/score-normalization.md]: Nit: It will -> They will

File: 2-api/clients.md (hunk 8 above)
Comment (rachelrosengoogle) [reply 2-api/clients.md:127]: This is still not resolved--I see attribute cards here, I only see the change in the other location.

File: 2-api/key-concepts.md [hunk 30]
```diff
@@ -2,71 +2,89 @@
...
 
-A comment label used for review. The most popular attributes are the toxicity and severe toxicity models.
+An **attribute** is a comment label used for review. The most popular attribute
+is `TOXICITY`.
+
+Attribute was previously known as "model". 
+
+#### Why did we make this name change?
+
+Machine Learning is a fast-moving discipline, with a lot of community-driven and historical terminalogy. It's our believe that the community has rough consensus on the term "machine learning model (ML model)". This term artifact, a neural net architecture together with the weights that result from training that neural net, defines something that can take inputs ("features") and produce outputs (class labels, regression scores, sometimes known as targets). Models generally can produce multiple outputs.
```
Comment (bfvitale) [inline 2-api/key-concepts.md]: change "believe" to "belief"

Comment (bfvitale) [inline 2-api/key-concepts.md]: "This term artifact" reads a little strange. Maybe a word got dropped?

File: 2-api/key-concepts.md (hunk 23 above)
Comment (alexandrascript) [reply 2-api/key-concepts.md:53]: Should I just remove the last sentence?

**Comment by alexandrascript** [comment]
Planning on merging by EOD, please let me know if there's any urgent reason this cannot happen.

**Review by rachelrosengoogle — APPROVED** [review]
